src/test/java/apoc/coll/CollTest.java

```diff
@@ -204,4 +205,138 @@ public void testSetOperations() throws Exception {
         testCall(db,"CALL apoc.coll.removeAll([1,2],[3,2])", r -> assertEquals(asList(1L),r.get("value")));
 
     }
+    
```

Comment:
Any suggestions and improvements for better testing indeterminate behavior are more than welcome.